src/box/box.cc

```diff
@@ -3550,8 +3550,13 @@ box_iproto_override(uint32_t req_type, iproto_handler_t handler,
 	return iproto_override(req_type, handler, destroy, ctx);
 }
 
-static inline void
-box_register_replica(uint32_t id, const struct tt_uuid *uuid)
+/**
+ * Register a new replica directly, bypassing all checks like whether the
+ * replica is writable. It makes the function usable by bootstrap master when
+ * it is read-only but has to make the first registration.
+ */
+static void
+box_register_new_replica_direct(uint32_t id, const struct tt_uuid *uuid)
```

Comment:
Nit: I'd rather call this function `box_insert_replica_record` or something like this to emphasize that it's a simple wrapper around `boxk(IPROTO_INSERT)`.

Reply:
Done. Although I wanted specifically to avoid any mentioning of records here, because didn't want the function name to depend on how we store replica rows and how many of them we store per replica. Anyway, can rename in the future again if needed.